EIPS/eip-7623.md

```diff
@@ -0,0 +1,89 @@
+---
+eip: 7623
+title: Increase calldata cost
+description: Increase calldata cost to decrease the maximum block size
+author: Toni Wahrstätter (@nerolation), Vitalik Buterin (@vbuterin)
+discussions-to: https://ethereum-magicians.org/t/eip-7623-increase-calldata-cost/18647
+status: Draft
+type: Standards Track
+category: Core
+created: 2024-02-13
+---
+
+
+
+## Abstract
+
+The current calldata pricing allows for significantly large blocks of up to 2.7 MB while the average block size is much smaller at 125 KB. 
+This EIP proposes an adjustment in the Ethereum calldata price to reduce the current maximum block size and its variance. 
+This is achieved through adjusting calldata pricing, aligning with emerging data storage practices in the network, especially in the context of [EIP-4844](./eip-4844.md).
+
+
+## Motivation
+
+The block gas limit has not been increased since [EIP-1559](./eip-1559.md), while the average size of blocks has continuously increased due to the growing number of rollups posting data to Layer 1. 
+[EIP-4844](./eip-4844.md) introduces blobs as a preferred method for data availability, signaling a shift away from calldata-dependent strategies. 
+This transition demands a reevaluation of calldata pricing, especially with regards to mitigating the inefficiency between the average block size in bytes and the maximum one possible.
+By increasing the gas cost for nonzero calldata bytes for transactions that are mainly using Ethereum for Data Availability (DA), the proposal aims to balance the need for block space with the necessity of reducing the maximum block size to make room for adding more blobs. The move from using calldata for DA to blobs further strengthens the multidimensional fee market by incentivizing blob space.
+
+Increasing calldata cost to 68 gas for transactions that do not spend more gas on EVM operations than a certain threshold, significantly reduces the maximum possible block size by lowering the number of huge, pure-DA transactions that fit into one block.
+However, the calldata price is kept at 16 gas per nonzero byte for those transactions that also spend enough on EVM computation, such that nothing changes for those users. Consequently, this approach safeguards users who depend on extensive calldata usage alongside EVM computations, ensuring they are not adversely affected by the new pricing model.
```

Comment:
These paragraphs should be in the Rationale section since they explain the choice of gas cost.

Reply:
Fixed